unassociated-license-ids.md

```diff
@@ -0,0 +1,47 @@
+# The following FSF license tags did not have any obvious match to an SPDX license
```

Comment:
I don't think we need this in version control, because you can generate it from `licenses.json` with `jq`.  Can we remove it here?  And if you like open an issue with a list of unchecked boxes?

Reply:
`jq` for this is:

    $ jq -r 'to_entries[] | select(.value.identifiers.spdx | not) | .key' licenses.json

Reply:
I'll replace it with an issue since some of the items have comments that would not be retained by using jq.